src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -397,7 +397,12 @@ private function createBlock($messages, $type = null, $style = null, $prefix = '
                 $message = OutputFormatter::escape($message);
             }
 
-            $lines = array_merge($lines, explode(PHP_EOL, wordwrap($message, $this->lineLength - $prefixLength - $indentLength, PHP_EOL, true)));
+            $length = $this->lineLength;
+            if (preg_match('(<\/.+>)', $message, $match)) {
```

Comment:
AFAIU, this hack only works in some specific cases. It won't work with something like `'Loading the <comment>configuration</comment> file "<comment>%s</comment>".',` for instance. At least, you should use `{<\/.+?>}` to avoid getting too much text. But even with this regex, I don't think this is the right way to fix the issue.

Reply:
I will probably break as well when using `<info>` first and `<comment>` at the end. Just another example that proves the fragility of this approach.

Reply:
To fix the issue, the entire algorithm should be changed IMHO.